com.github.nostar.DroidStar.metainfo.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.github.nostar.DroidStar</id>
```

Comment:
as I mentionned in the PR you close, the app id should be starting with `io.github` as per the documentation.